### ISSUE TITLE
Issue with jQuery.captures and Page type confirmations

### ISSUE THREAD

**Issue by stuporglue** [kodie/gravityforms-repeater#12]
I've got a series of forms that are chained together. It's a several-hundred question scholarship application process, so each section is a form. Upon completion of one form the user is sent directly to the next form using the Gravity Forms Page type confirmation.

It seems that at least in a page type confirmation the jQuery.captures data is not cleared so values typed in one form overwrite values in a subsequent form if the field names match. 

I have worked around this issue by appending an additional hidden field to the form which holds the formId. During initialization if the jQuery.captures formId doesn't match the current form ID, gfRepeater_submitted is not set to true. 

```
// Initiation right away
jQuery(document).ready(function($) {
    gfRepeater_patchMask();
    var theForm = jQuery('.gform_wrapper form');
    var formId = jQuery('.gform_wrapper form').attr('id');
    theForm.append('<input type="hidden" name="the_hidden_form_id" value="' + formId + '">');
    theForm.capture();
    if (jQuery.captures()) { 
        // check if it's the same form
        if(jQuery.captures('the_hidden_form_id') == formId){
            gfRepeater_submitted = true; 
        }else{
            // console.log("Found old data, not counting as submitted");
        }
    }
});
```

**Comment by kodie** [comment]
Thank you for the information and the suggestion! I will get this fixed in the next development release!

**Comment by kodie** [comment]
Hey Michael! I have fixed this in the 1742041 commit! Test it out by installing the latest development version.

Find out how to get the latest development version here: https://github.com/kodie/gravityforms-repeater#development
